pymess/backend/dialer/daktela.py

```diff
@@ -79,7 +87,8 @@ def publish_message(self, message):
         client_url = self._get_dialer_api_url()
         try:
             payload = {
-                'queue': settings.DIALER_DAKTELA.QUEUE,
+                'queue': (settings.DIALER_DAKTELA.PREDICTIVE_DIALER_QUEUE if message.is_predictive
```

Comment:
MANUAL_QUEUE

Reply:
stále ten název nedává smysl. To slovo dialer je blbost máme AUTODIALER_QUEUE tak toto může být maximálně PREDICTIVE_QUEUE

Reply:
changed to PREDICTIVE_QUEUE